### ISSUE TITLE
Create convenience constructor for FormField

### ISSUE THREAD

**Issue by cmpaul** [HelloFax/hellosign-java-sdk#52]
This is a bit verbose:

```java
            FormField textField = new FormField();
            textField.setSigner(0);
            textField.setApiId("textfield_1");
            textField.setHeight(25);
            textField.setWidth(300);
            textField.setName("Name");
            textField.setPage(1);
            textField.setIsRequired(true);
            textField.setType(FieldType.text);
            textField.setValidationType(ValidationType.letters_only);
            textField.setX(100);
            textField.setY(100);
```

Let's add some constructor options to take the minimum required parameters:

* Type
* Signer
* Name (Field Label)
* x, y coordinates
* width, height

(this wouldn't be *less* verbose, but it would be indicative of what's required at least, which isn't real clear right now)

**Comment by mveitas** [comment]
Something to consider in the future is using the builder pattern to help construct these objects. When you make the switch to Java 8, you can even make the objects self documenting by making optional objects immutable. We have started using https://immutables.github.io/ and it has helped out with a lot of boilerplate code.

**Comment by cmpaul** [comment]
@mveitas Thanks for the tip! Any chance you can provide an example of how this might look?

**Comment by cmpaul** [comment]
Released as part of v4.0.4
